gr-blocks/python/blocks/pdu_compatibility.py

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright 2021 Free Software Foundation, Inc.
+#
+# This file is part of GNU Radio
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+#
+
+'''
+Compatibility layer for transition to gr-pdu. Scheduled for removal in 3.11.
+'''
+
+
+from gnuradio import pdu
+
+byte_t = pdu.types.byte_t
+float_t = pdu.types.float_t
+complex_t = pdu.types.complex_t
+vector_type = pdu.types.vector_type
+
+class pdu_filter(pdu.pdu_filter):
+    def __init__(self, *args, **kwargs):
+        print("WARNING: `pdu_filter` has moved to gr-pdu and will be removed from gr-blocks soon. Please update to use pdu.pdu_filter()")
```

Comment:
I assume this should go to `gr.log.warn`.

Reply:
Agree